redcycle/migrations/0001_initial.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 3.1.3 on 2020-11-15 23:10
```

Comment:
Whenever you make changes to the models and apply the changes, Django generates "migration" files to implement the changes. This is one of them. Not really useful to look at right now, but they should be committed to git